iroh-dns-server/src/metrics.rs

```diff
@@ -9,7 +9,7 @@ use struct_iterable::Iterable;
 pub struct Metrics {
     pub pkarr_publish_update: Counter,
     pub pkarr_publish_noop: Counter,
-    pub pkarr_publish_error: Counter,
+    // pub pkarr_publish_error: Counter,
```

Comment:
stray comment, plz remove

Reply:
The point of this PR was for the team to claim which ones they would like to keep as they are unused. I'm going to nuke all the ones people didn't vote to keep.